contracts/Election.sol

```diff
@@ -96,24 +96,24 @@ contract Election {
         address highestCandidate;
         for (uint x = 0; x < numOfCandidates; x++) {
             address currentAddr = candiatesList[x];
-            Candidate storage currentCandidate = candidateData[round][currentAddr];
+            Candidate memory currentCandidate = candidateData[round][currentAddr];
             if (currentCandidate.vote > maxVote) {
                 highestCandidate = currentAddr;
                 maxVote = currentCandidate.vote;
             }
         }
-        Candidate storage electedPerson = candidateData[round][highestCandidate];
+        Candidate memory electedPerson = candidateData[round][highestCandidate];
```

Comment:
same problem here

Reply:
oh!!!! got it! thanks, let me change it back!